x11-wm/spectrwm/spectrwm-3.5.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit toolchain-funcs
+
+DESCRIPTION="Small dynamic tiling window manager for X11"
+HOMEPAGE="https://github.com/conformal/spectrwm"
+SRC_URI="https://github.com/conformal/spectrwm/archive/${PN^^}_${PV//./_}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	elibc_musl? ( sys-libs/queue-standalone )
+	x11-libs/libxcb
+	x11-libs/libX11
+	x11-libs/libXcursor
+	x11-libs/libXft
+	x11-libs/libXrandr
+	x11-libs/xcb-util
+	x11-libs/xcb-util-keysyms
+	x11-libs/xcb-util-wm
+"
+RDEPEND="
+	${DEPEND}
+	x11-misc/dmenu
+"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/spectrwm-3.5.0-fix-cflags-ldflags.patch
+)
+S=${WORKDIR}/${PN}-${PN^^}_${PV//./_}
```

Comment:
could you put `S` below `SRC_URI` and also add quotes.

Reply:
done, thanks.